scene/main/viewport.cpp

```diff
@@ -2339,7 +2339,7 @@ void Viewport::_gui_force_drag(Control *p_base, const Variant &p_data, Control *
 
 void Viewport::_gui_set_drag_preview(Control *p_base, Control *p_control) {
 	ERR_FAIL_NULL(p_control);
-	ERR_FAIL_COND(!Object::cast_to<Control>((Object *)p_control));
+	ERR_FAIL_NULL(Object::cast_to<Control>((Object *)p_control));
```

Comment:
This line is technically redundant, but left it in, can remove it if desired, as the pointer is already `Control` this check should be redundant

Reply:
Didn't dig into the blame but the only way this makes sense would be if the argument used to be a `Variant`